test/presenters/bill-runs/two-part-tariff/review-licence.presenter.test.js

```diff
@@ -186,6 +187,28 @@ describe('Review Licence presenter', () => {
         })
       })
     })
+
+    describe("the 'adjustments' property", () => {
+      describe('when a charge reference has an aggregate or charge factor adjustment', () => {
+        it("changes the chargeReferenceLink to 'Change details'", () => {
+          const result = ReviewLicencePresenter.go(billRun, licence)
+
+          expect(result.chargeData[0].chargeReferences[0].chargeReferenceLink.linkName).to.equal('Change details')
+        })
+      })
+
+      describe('when a charge reference does not have an aggregate or charge factor adjustment', () => {
+        beforeEach(() => {
+          licence[0].reviewChargeVersions[0].reviewChargeReferences[0].chargeReference.adjustments = { charge: null, aggregate: null }
+        })
+
+        it("changes the chargeReferenceLink to 'View details'", () => {
+          const result = ReviewLicencePresenter.go(billRun, licence)
+
+          expect(result.chargeData[0].chargeReferences[0].chargeReferenceLink.linkName).to.equal('View details')
+        })
+      })
```

Comment:
I'm guessing if you look at your code coverage it is going to say you are missing some on line 133 in the presenter.

It's only one more test that can be created by copying & pasting another. So, I would suggest having both `charge: 0.5` and `aggregate: 0.5` being tested.